distributed/protocol/tests/test_torch.py

```diff
@@ -18,9 +18,10 @@ def test_grad():
     x = np.arange(10)
     t = torch.Tensor(x)
     t.grad = torch.zeros_like(t) + 1
+    t.requires_grad = True
```

Comment:
It'd be nice if this test could have both values of `requires_grad`. Maybe `pytest.mark.parametrize`?

Reply:
I don't know how to do that :S I can investigate though.

Reply:
Something like

``` python
@pytest.mark.parametrize("requires_grad", [True, False])
def test_grad(requires_grad):
    x = torch.zeros(..., requires_grad=requires_grad)
```

Reply:
Very cool. I did not know about this. It is the first time I am using pytest. I have done something similar.